tensorflow/lite/tools/versioning/op_version.cc

```diff
@@ -487,6 +509,53 @@ OpSignature GetOpSignature(const OperatorCode* op_code, const Operator* op,
       }
     } break;
 
+    case BuiltinOperator_ADD:
+    case BuiltinOperator_SUB: {
+      op_sig.options.addsub.pot_scale_int16 = false;
+      const Tensor* input1_tensor =
```

Comment:
Hi @suharshs and @jdduke !
I looked into schema.fbs and it looks they if I add a new parameter, then a lot of places should be changed.
I found this approach to differ between two cases: general parameter scale and pot param scale, and set different version accordingly.
Could you please take a look ?
If this approach is wrong, then I will try to add a new option to schema.fbs for ADD/SUB operators.
Thanks!

Reply:
Hi, something needs to set the pot_scale_int16 so that versioning can make the right decision. This info has to be in the flatbuffer and then converter to the c api so that the runtime can make the right kernel choice as well.

Here are instructions that may be helpful outline the changes that need to be made: https://www.tensorflow.org/lite/guide/ops_version

Reply:
Thanks! The link is very helpful. I will correct.

Reply:
Hi @suharshs ! I have a quick question (this is not mentioned in the link above):

Once I changed schema.fbs file, it looks that I need to re-generate the file schema_generated.h (which is not re-generated automatically by bazel).
I use this command:

flatc -c schema.fbs

The generated file is very different to the file commited to the repo.
Which command should I call to generate a new schema_generated.h for my new options ?

Thanks for the help!

Reply:
Hi Elena, checking with folks on the team about what the correct commands are and will get back to you.

Reply:
Hi Suharsh, 
Currently I modified it manually (maybe, it is enough) and pushed a new implementation.
Could you please take a look ?
Thanks!

Reply:
Hi @suharsh, 
The document in the link above has some out-of-state  filenames. 
I updated them and here is a PR, probably, for another team to review:
https://github.com/tensorflow/tensorflow/pull/38201